src/adafruit_blinka/microcontroller/mcp2221/mcp2221.py

```diff
@@ -396,7 +474,7 @@ def dac_write(self, pin, value):
         report[4] = 1 << 7 | (value & 0b11111)
         self._hid_xfer(report)
 
-    # pylint: enable=unused-argument
+   # pylint: enable=unused-argument
 
 
 mcp2221 = MCP2221()
```

Comment:
As mentioned in #496, this var is problematic for different reasons:
* if we remove it, people that are importing it will get an `ImportError` (unless it's considered private and can be removed);
* if we leave it, it will always open and lock the first MCP device even if it's not needed (e.g. if only the second MCP is needed), and this will prevent other processes to open and use it.

Reply:
This is now fixed by using `__getattr__`.